config/routes.rb

```diff
@@ -2,6 +2,12 @@
   devise_for :users
 
   namespace :editorial do
+    get '/news' => 'news#index', as: :news_articles
```

Comment:
Oh so you can namespace things in forms:

    simple_form_for [ :editorial, @news_article ]

Or worst case specify the URL directly:

    simple_form_for @news_article, url: whatever_path